packages/shared/lib/db/migrations/20240529180658_add_notifications_table_index.cjs

```diff
@@ -0,0 +1,9 @@
+exports.config = { transaction: false };
+
+exports.up = async function (knex, _) {
+    await knex.schema.raw('CREATE INDEX CONCURRENTLY "idx_sync_id_active_true" ON "_nango_ui_notifications" USING BTREE ("sync_id") WHERE active = true');
```

Comment:
Didn’t want to over optimize but this is added with the query in the sync.service in mind